__tests__/app.test.js

```diff
@@ -168,3 +168,85 @@ describe("/api/articles/:article_id/comments", () => {
       });
   });
 });
+
+describe.only("POST /api/articles/:article_id/comments", () => {
```

Comment:
A couple more test cases to consider 

When posting a comment, are unnecessary properties ignored? 
eg {
    username: Beth 
    body: "comment"
    isCool: true
}

Also, what happens when a required field is missing? 
eg 
{
    body: "comment"
}